manim/mobject/mobject.py

```diff
@@ -1804,15 +1847,22 @@ def update_target(*method_args, **method_kwargs):
             return self
 
         self.is_chaining = True
+        self.cannot_call = True
+
         return update_target
 
     def build(self):
         from ..animation.transform import _MethodAnimation
 
         if self.overridden_animation:
-            return self.overridden_animation
+            anim = self.overridden_animation
+        else:
+            anim = _MethodAnimation(self.mobject, self.methods)
+
+        for attr, value in self.anim_args.items():
+            setattr(anim, attr, value)
```

Comment:
Wouldn't it be better to pass `self.anim_args`(probably rename it to `self.anim_kwargs`) directly into `_MethodAnimation` like 
```py 
anim = _MethodAnimation(self.mobject, self.methods, **self.anim_args)
``` 
instead of the for loop? (`_MethodAnimation` currently doesn't accept `kwargs` but it's just a matter of adding `**kwargs` into `_MethodAnimation`'s constructor and passing it upstream)

Reply:
I thought about it, but I decided the current way was better so the same logic could be used for normal and overridden animations, and setting the arguments as attributes is the same thing that `Scene.play` does.

As for the rename to `anim_kwargs`, I'd be okay with it, but personally I find `anim_args` descriptive enough.

Reply:
> he same logic could be used for normal and overridden animations, and setting the arguments as attributes is the same thing that `Scene.play` does

I see, that makes sense